mdoc/Mono.Documentation/Updater/Statistics/StatisticsCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Mono.Documentation.Updater.Frameworks;
+
+namespace Mono.Documentation.Updater.Statistics
+{
+    public class StatisticsCollector
+    {
+        public Dictionary<string, StatisticsStorage> Storages { get; } =
```

Comment:
Please include some triple slash documentation to make it clear, for example, what the `string` key represents